dist/s_string.ok

```diff
@@ -802,7 +802,6 @@ hashval
 havesize
 hdr
 hhh
-highjack
```

Comment:
Awww. I like that word :)

Reply:
It's still there! It was just spelled incorrectly, I fixed it so now it doesn't need to be in this list.